onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -527,6 +528,24 @@ TEST_F(GraphTransformationTests, FuseConvBNMulAddUnsqueeze) {
   }
 }
 
+TEST_F(GraphTransformationTests, NotWhereFusion) {
```

Comment:
Are there any counter examples to test. For example, Not node output is also a graph output. The code does not fuse Not node and Where node.

Reply:
added to the test graph